content/post/44-ipld-explorer-cli-0.14.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2018-08-30
+url: 44-ipld-explorer-cli-0-14
+title: ipld-explorer-cli 0.14 released
+author: Alan Shaw
+---
+
+There’s a new version of the CLI tool for exploring your DAGs!
```

Comment:
Blog readers will have less background knowledge, so let's move more of it up front. For example:

> There's a new version of the CLI tool for exploring your IPLD DAGs! Let's break down those acronyms and explain why I'm so excited about this.

// then you can pull up line 14

> [IPLD (Inter Planetary Linked Data)](https://ipld.io/) is the underlying data structure used by IPFS that allows it to store, retrieve and traverse through any type of data. Recently, our team created the awesome new [IPLD Explorer](https://explore.ipld.io) so you can visually explore IPLD objects. If you haven't already checked it out, yhop over and explore everything from [git repos](https://explore.ipld.io/#/explore/z8mWaJHXieAVxxLagBpdaNWFEBKVWmMiE) to [ethereum blocks](https://explore.ipld.io/#/explore/z43AaGEvwdfzjrCZ3Sq7DKxdDHrwoaPQDtqF4jfdkNEVTiqGVFW) to [historic photo archives of the lunar landings](https://explore.ipld.io/#/explore/QmSnuWmxptJZdLJpKRarxBMS2Ju2oANVrgbr2xWbie9b2D) to your own IPLD data. 

The IPLD Explorer also comes with a command-line tool ([ipld-explorer-cli](https://www.npmjs.com/package/ipld-explorer-cli)) so you can explore with your keyboard. I was inspired to give it an upgrade so that it could resolve IPLD formats other than..."

//then video